### PR TITLE
Upgarde to scala-landsat-util 1.0.0, GeoTrellis 1.0.0, and akka-http

### PR DESCRIPTION
**Pull request by lossyrob**
This PR upgrades to `scala-landsat-util` 1.0.0, `geotrellis` 1.0.0, and replaces the spray usage with akka-http.

To test locally, run the local ingest, tile server and viewer as described in the README.

### CODE REVIEW

**Comment by lossyrob** [comment]
@dorukozturk can you test to make sure this works for your use case? Thank you!

**Comment by dorukozturk** [comment]
@lossyrob I will test this and let you know as soon as possible !!! Thanks.

**Comment by pomadchin** [comment]
@lossyrob improved shading: 

```scala
assemblyShadeRules in assembly := {
  val shadePackage = "com.azavea.shaded.demo"
  Seq(
    ShadeRule.rename("com.google.common.**" -> s"$shadePackage.google.common.@1")
      .inLibrary(
        "com.azavea.geotrellis" %% "geotrellis-cassandra" % gtVersion,
        "com.github.fge" % "json-schema-validator" % "2.2.6",
        "com.google.code.gson" % "gson" % "2.3.1",
        "com.google.jimfs" % "jimfs" % "1.1"
      ).inProject
  )
}
```

**Comment by pomadchin** [comment]
@lossyrob instead of shading (as it breaks akka) we can use: 

```scala
dependencyOverrides += "com.google.guava" % "guava" % "16.0.1"
```

File: ingest/build.sbt [hunk 1]
```diff
@@ -24,13 +24,13 @@ assemblyMergeStrategy in assembly := {
   case _ => MergeStrategy.first
```
Comment (pomadchin) [inline ingest/build.sbt]: mb [this](https://github.com/pomadchin/geotrellis-chatta-demo/blob/168cd59f4c066aa541b199a33e1bf4f21e5581b0/geotrellis/build.sbt#L37-L42) way to define assembly strategy is more compact?

**Comment by pomadchin** [comment]
@lossyrob About verify error https://github.com/geotrellis/geotrellis-chatta-demo/pull/29
in short: workaround to use bad `-noverify` flag

**Comment by dorukozturk** [comment]
This PR worked for our use case. Thanks !!!

**Pull request merged**